Thank you for @archiewood, See the CI failed with linter error: https://github.com/arshad-yaseen/monacopilot/actions/runs/13935782132/job/39012950991?pr=133#step:6:33

```
Error: packages/core build: src/logger.ts(1,7): error TS6133: 'RED' is declared but its value is never read.
Error: packages/core build: src/logger.ts(2,7): error TS6133: 'YELLOW' is declared but its value is never read.
```

--- a/packages/core/src/llm/providers/mistral.ts
+++ b/packages/core/src/llm/providers/mistral.ts
@@ -50,7 +50,7 @@ export class MistralHandler extends BaseProviderHandler<'mistral'> {
         return Array.isArray(content)
             ? content
                   .filter(chunk => 'text' in chunk)
-                  .map(chunk => chunk.text)
+                  .map(chunk => (chunk as { text: string }).text)
                   .join('')
             : content;
     }

--- a/packages/core/src/logger.ts
+++ b/packages/core/src/logger.ts
@@ -1,5 +1,3 @@
-const RED = '\x1b[91m';
-const YELLOW = '\x1b[93m';
 const RESET = '\x1b[0m';
 const BOLD = '\x1b[1m';
 

